Network February release

anton-evseev opened the pull request:
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>

Please merged https://github.com/Azure/azure-rest-api-specs/pull/5223 first

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

MikhailTryakhov committed:
Adds base for updating Microsoft.Network from version stable/2018-12-01 to version 2018-12-01

MikhailTryakhov committed:
Updates readme

MikhailTryakhov committed:
Updates API version in new specs and examples

sergey-shandar committed:
Network: Duplicate tags in readme file. (#5227)

sergey-shandar committed:
FI from `master` branch to `network-february-release`. (#5262)

* DataFactory Swagger Change for SapOpenHub and Rest (#5195)

* DataFactory Swagger Change for SapOpenHub and Rest

* correct indentation

* address comment of REST

* fix syntax issue

* Fix issue with AutoRest

* Add cognitive service AnomalyFinder (#5126)

* add AnomalyFinder swagger json

* AnomalyFinder update swagger json

* add AnomalyFinder readme

* fix file path; change class name;

* make camelCase style

* fix typo; change operationId; remove hard code arg limitation;

* fix type of "sensitivity"; change description of "timestamp";

* Improving secretVersion description (#5220)

* [Cogsvcs] LUIS - add text to simple entity description (#5216)

* Add hwRevision field to HanaInstance definition (#5196)

* [DataFactory] Swagger changes for ADF newly released features: (#5218)

* Swagger changes for ADF newly released features:
1. I can load data from o365 by leveraging the REST endpoint of o365
2. Support copying incremental files based on lastModifiedTime for S3/File/Blob
3. Enable ADLS Gen2 connector as both source & sink
4. Support data migration from MongoDB to CosmosDB (MongoDB API)
5. Make Kusto source/sink better and support Azure Data exploer source/sink
6. Support copy from ADLS Gen1 to ADLS Gen2 with ACL
7. serviceUrl @ S3 linked service
8. I can copy from Dynamics AX
9. AAD OAuth for OData Linked Service
10. I can copy from Oracle Service Cloud and Google Adwords

* fix swagger definetion for MongoDB Srouce cursor method.

* fix Amazon S3 servie url definition

* fix CosmosDbMongoDbApiSink

* Fix several typos

* Add maxConcurrentConnections in CopySource and CopySink

* Node v 10. (#5236)

* Remove API reference of Gender::genderless. (#5188)

* Add an Operations API for Commitment Plan RP (#5222)

* Add an Operations API for Commitment Plan RP

* Add OperationEntityListResult related definitions

* Fix case typo

* Fix tab to spaces

* Fix NewtonSoftJson Issue (#5240)

* Support create pipeline run with recovery mode

* Update description of isLatest

* Update descriptions

* update autorest version to fix CI

* Add suppresses for Machine Learning Commitment Plan RP (#5221)

* typo: datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/ (#5238)

- Adwords -> AdWords
- connectioned to -> for the

* scripts (#5213)

* scripts

* TRAVIS_PULL_REQUEST

* another try

* TRAVIS_PULL_REQUEST_SLUG

* TRAVIS_REPO_SLUG

* Swagger to SDK

* linter and ruby

* not really a slug

* pool

* indent

* strange

* URI

* env vars

* uri

* env var

* TRAVIS_PULL_REQUEST_SLUG

* matrix

* Azure OpenAPI name

* linter: autorest fix

* error handling and PR_ONLY = true

* syntax

* pipelines.

* job name

* job name

* Names

* params, no getStatus.js

* No CI, only PR validations.

* displayName

* revert back

* no need to show vars

* Fix for lintdiff tool memory issue. (#5246)

* node options

* correct NODE_OPTIONS

* lintdiff travis-ci

* 2019 Azure Stack hybrid Profile. (#5212)

* 2019 Azure Stack hybrid Profile.

* fixing changes for profile

* remove space

* formatted profile

* [Azure Search] Add Suggest and Index APIs to data plane Swagger spec (#5219)

* Renaming Search|SuggestParametersPayload to *Request and making them internal

Made AutocompleteRequest internal too, and fixed some incorrect comments.

* Adding Suggest API to the Azure Search data plane spec

* Fixing validation error in Suggest example

* Adding Index API to the Azure Search data plane spec

* Make example names consistent

* Adding new properties to Databricks LS and custom activity (#5214)

* typo: instantitate -> instantiate (#5251)

* Users/laramume/addv2.1 swagger (#5187)

* updated model names in preview version to be consistent with stable version to avoid breaking changes in client side code

* changed swagger files

* added v2.1 swagger

* added v2.1 swagger

* added v2.1 swagger

* added v2.1 swagger

* added v2.1 swagger

* resolved PR comments

* Update swagger file and examples with time zone parameter (#5224)

* Update swagger documetation to mark Collation property as writable.

Change Managed Instance collation property from read only to read and create.
Add collation to properties section in ManagedInstanceCreateMax example.

VSTS task: 12336257

* Remove 'cosmetics' from swagger.

* Remove comma as redundant

* Return to the initial, generated state.

* Update swagger file and examples with time zone parameter

* Minor (but improtant) changes in Azure DevOps pipeline. (#5253)

* empty parameters

* TRAVIS_BRANCH

* target branch

* trying to set env var

* use shell

* info

* more info

* Pull Request Number

* [Ready to MERGE] Swagger for Tenant Properties API (#5038)

* added swagger for Tenant Properties

added swagger for Tenant Properties

* Fixed camel casing in billing.json file

Fixed camel casing in billing.json file

* Update billing.json

* changes in DiscoverTenants contracts

changes in DiscoverTenants contracts

* billing.Json trim blank space

billing.Json trim blank space

* Update billing.json

* removal of plural name

removal of plural name

* Updating the route

Updating the route as per suggestion

* Moved Swagger for Discover Tenants from billing to consumption

Moved Swagger for Discover Tenants from billing to consumption

* Change in Namespace followed by one additional parameter "billingAccountId"

Change in Namespace followed by one additional parameter "billingAccountId"

* Change of method name from discoverTenants to tanants.

Change of method name from discoverTenants to tanants.

* changes ion response type

changes ion response type

* added name after Id. Code review implementation

added name after Id. Code review implementation

* Add JavaScript/TypeScript generation details to resource-manager/netapp (#5261)

* Azure Security Center - break-to-composite v1 (#5197)

* Azure Security Center - break-to-composite v1

* Azure Security Center - composite api v1 - suppress email format

sergey-shandar committed:
Merge branch 'network-february-release' into sergey/fi

sergey-shandar committed:
Merge pull request #5275 from Azure/sergey/fi

FI to network

syfarogh committed:
Add globalReachEnabled, Add Get and list for peer express route circuit connections. Remove allowGlobalReach (#5266)

* Added GlobalReachEnabled field. Added GET and List apis for peer express route circuit connections. Removed AllowGlobalReach as it was a dead field and abandoned in favor for GlobalReachEnabled

* adding allow gloabl reach back

* fixed typo

anton-evseev committed:
Fix descriptions & remove unused/duplicate definitions (#5243)

raraveen committed:
Add support for Local Sku (#5317)

azhao155 committed:
Add WAF as a top level resource (#5341)

* Add WAF as a top level resource

* Fix merge

* Addressed comments

* Addressed comments

* Code review comments

* fix issue for powershell

* add health probe log

* address the comments

* increase Node.js memory

* double the memory

* print node_options

* mistake

* pass to docker

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-python
<details><summary>Encountered a Subprocess error: (azure-sdk-for-python)</summary><p>

Command: ['/usr/local/bin/autorest', '/tmp/tmpdxq3y2tk/rest/specification/network/resource-manager/readme.md', '--multiapi', '--python', '--python-mode=update', '--python-sdks-folder=/tmp/tmpdxq3y2tk/sdk', '--use=@microsoft.azure/autorest.python@~3.0.56', '--version=preview']
Finished with return code -6
and output:
```shell
AutoRest code generation utility [version: 2.0.4283; node: v8.11.3]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/root/.autorest/@microsoft.azure_autorest-core@2.0.4302/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4302)
   Loading AutoRest extension '@microsoft.azure/autorest.python' (~3.0.56->3.0.58)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.44->2.3.44)
Processing batch task - {"tag":"package-2018-12"} .

<--- Last few GCs --->

[20864:0x2aade60]    98414 ms: Mark-sweep 1406.8 (1481.4) -> 1402.7 (1477.9) MB, 1699.6 / 0.0 ms  allocation failure GC in old space requested
[20864:0x2aade60]    99635 ms: Mark-sweep 1402.7 (1477.9) -> 1402.7 (1446.9) MB, 1221.6 / 0.0 ms  last resort GC in old space requested
[20864:0x2aade60]   101092 ms: Mark-sweep 1402.7 (1446.9) -> 1402.7 (1446.9) MB, 1456.4 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x2e6d59d25879 <JSObject>
    1: add [native weak-collection.js:~64] [pc=0x73c9e724d29](this=0x73251cbcac1 <JSWeakSet>,j=0xcfc41710529 <JSProxy>)
    2: losslessJsonSerializable(aka losslessJsonSerializable) [/root/.autorest/@microsoft.azure_autorest-core@2.0.4302/node_modules/@microsoft.azure/autorest-core/dist/lib/ref/yaml.js:~232] [pc=0x73c9e55913b](this=0x32063ff822d1 <undefined>,o=0xcfc41710529 <JSProxy>)
    3: arg...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: node::Abort() [node]
 2: 0x8c21ec [node]
 3: v8::Utils::ReportOOMFailure(char const*, bool) [node]
 4: v8::internal::V8::FatalProcessOutOfMemory(char const*, bool) [node]
 5: v8::internal::Factory::NewFixedArray(int, v8::internal::PretenureFlag) [node]
 6: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::NewInternal(v8::internal::Isolate*, int, v8::internal::PretenureFlag) [node]
 7: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::New(v8::internal::Isolate*, int, v8::internal::PretenureFlag, v8::internal::MinimumCapacity) [node]
 8: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::EnsureCapacity(v8::internal::Handle<v8::internal::ObjectHashTable>, int, v8::internal::PretenureFlag) [node]
 9: v8::internal::ObjectHashTable::Put(v8::internal::Handle<v8::internal::ObjectHashTable>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int) [node]
10: v8::internal::JSWeakCollection::Set(v8::internal::Handle<v8::internal::JSWeakCollection>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int) [node]
11: v8::internal::Runtime_WeakCollectionSet(int, v8::internal::Object**, v8::internal::Isolate*) [node]
12: 0x73c9e4042fd
```

</p></details>

AutorestCI commented:
# Automation for azure-sdk-for-js
<details><summary>Encountered a Subprocess error: (azure-sdk-for-js)</summary><p>

Command: ['/usr/local/bin/autorest', '/tmp/tmplfwrr5vm/rest/specification/network/resource-manager/readme.md', '--license-header=MICROSOFT_MIT_NO_VERSION', '--typescript', '--typescript-sdks-folder=/tmp/tmplfwrr5vm/azure-sdk-for-js', '--use=@microsoft.azure/autorest.typescript@2.1.1']
Finished with return code -6
and output:
```shell
AutoRest code generation utility [version: 2.0.4283; node: v8.12.0]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/root/.autorest/@microsoft.azure_autorest-core@2.0.4299/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4299)
   Loading AutoRest extension '@microsoft.azure/autorest.typescript' (2.1.1->2.1.1)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.51->2.3.51)

<--- Last few GCs --->

[25338:0x3deb860]   106450 ms: Mark-sweep 1400.3 (1477.9) -> 1400.2 (1478.4) MB, 1879.6 / 0.0 ms  allocation failure GC in old space requested
[25338:0x3deb860]   108083 ms: Mark-sweep 1400.2 (1478.4) -> 1400.2 (1443.9) MB, 1633.2 / 0.0 ms  last resort GC in old space requested
[25338:0x3deb860]   109722 ms: Mark-sweep 1400.2 (1443.9) -> 1400.2 (1443.9) MB, 1638.6 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x363d5aca5879 <JSObject>
    1: add [native weak-collection.js:~64] [pc=0x1d9582b7c7c9](this=0x8a02dfd75e1 <JSWeakSet>,j=0x2e96c19bef21 <JSProxy>)
    2: losslessJsonSerializable(aka losslessJsonSerializable) [/root/.autorest/@microsoft.azure_autorest-core@2.0.4299/node_modules/@microsoft.azure/autorest-core/dist/lib/ref/yaml.js:~232] [pc=0x1d9582afbb7b](this=0x3a00c60822d1 <undefined>,o=0x2e96c19bef21 <JSProxy>)
    3:...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: node::Abort() [node]
 2: 0x8cbf4c [node]
 3: v8::Utils::ReportOOMFailure(char const*, bool) [node]
 4: v8::internal::V8::FatalProcessOutOfMemory(char const*, bool) [node]
 5: v8::internal::Factory::NewFixedArray(int, v8::internal::PretenureFlag) [node]
 6: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::NewInternal(v8::internal::Isolate*, int, v8::internal::PretenureFlag) [node]
 7: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::New(v8::internal::Isolate*, int, v8::internal::PretenureFlag, v8::internal::MinimumCapacity) [node]
 8: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::EnsureCapacity(v8::internal::Handle<v8::internal::ObjectHashTable>, int, v8::internal::PretenureFlag) [node]
 9: v8::internal::ObjectHashTable::Put(v8::internal::Handle<v8::internal::ObjectHashTable>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int) [node]
10: v8::internal::JSWeakCollection::Set(v8::internal::Handle<v8::internal::JSWeakCollection>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int) [node]
11: v8::internal::Runtime_WeakCollectionSet(int, v8::internal::Object**, v8::internal::Isolate*) [node]
12: 0x1d95829842fd
```

</p></details>

AutorestCI commented:
# Automation for azure-sdk-for-node
<details><summary>Encountered a Subprocess error: (azure-sdk-for-node)</summary><p>

Command: ['/usr/local/bin/autorest', '/tmp/tmpur5hw1av/rest/specification/network/resource-manager/readme.md', '--license-header=MICROSOFT_MIT_NO_VERSION', '--node-sdks-folder=/tmp/tmpur5hw1av/azure-sdk-for-node', '--nodejs', '--use=@microsoft.azure/autorest.nodejs@2.2.131']
Finished with return code -6
and output:
```shell
AutoRest code generation utility [version: 2.0.4283; node: v8.12.0]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/root/.autorest/@microsoft.azure_autorest-core@2.0.4289/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4289)
   Loading AutoRest extension '@microsoft.azure/autorest.nodejs' (2.2.131->2.2.131)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.44->2.3.44)

<--- Last few GCs --->

[20930:0x3511860]   153075 ms: Mark-sweep 1399.7 (1476.9) -> 1399.6 (1477.4) MB, 2288.2 / 0.0 ms  allocation failure GC in old space requested
[20930:0x3511860]   155255 ms: Mark-sweep 1399.6 (1477.4) -> 1399.6 (1442.9) MB, 2179.8 / 0.0 ms  last resort GC in old space requested
[20930:0x3511860]   158008 ms: Mark-sweep 1399.6 (1442.9) -> 1399.6 (1441.9) MB, 2752.3 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x260660da5879 <JSObject>
    1: add [native weak-collection.js:~64] [pc=0x33ce85b318c9](this=0x27bd4127f2a9 <JSWeakSet>,j=0x2da2756657b9 <JSProxy>)
    2: losslessJsonSerializable(aka losslessJsonSerializable) [/root/.autorest/@microsoft.azure_autorest-core@2.0.4289/node_modules/@microsoft.azure/autorest-core/dist/lib/ref/yaml.js:~232] [pc=0x33ce85b5a93b](this=0x31cd5be022d1 <undefined>,o=0x2da2756657b9 <JSProxy>)
    3...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: node::Abort() [node]
 2: 0x8cbf4c [node]
 3: v8::Utils::ReportOOMFailure(char const*, bool) [node]
 4: v8::internal::V8::FatalProcessOutOfMemory(char const*, bool) [node]
 5: v8::internal::Factory::NewFixedArray(int, v8::internal::PretenureFlag) [node]
 6: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::NewInternal(v8::internal::Isolate*, int, v8::internal::PretenureFlag) [node]
 7: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::New(v8::internal::Isolate*, int, v8::internal::PretenureFlag, v8::internal::MinimumCapacity) [node]
 8: v8::internal::HashTable<v8::internal::ObjectHashTable, v8::internal::ObjectHashTableShape>::EnsureCapacity(v8::internal::Handle<v8::internal::ObjectHashTable>, int, v8::internal::PretenureFlag) [node]
 9: v8::internal::ObjectHashTable::Put(v8::internal::Handle<v8::internal::ObjectHashTable>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int) [node]
10: v8::internal::JSWeakCollection::Set(v8::internal::Handle<v8::internal::JSWeakCollection>, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int) [node]
11: v8::internal::Runtime_WeakCollectionSet(int, v8::internal::Object**, v8::internal::Isolate*) [node]
12: 0x33ce858042fd
```

</p></details>

AutorestCI commented:
# Automation for azure-sdk-for-ruby
A PR has been created for you:
https://github.com/Azure/azure-sdk-for-ruby/pull/2314

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

AutorestCI commented:
# Automation for azure-sdk-for-go

AutorestCI commented:
# Automation for azure-sdk-for-java

number213 commented:
"SDK Generation - Python" generation keeps failing with OOM error

dsgouda commented:
startbuild

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

dsgouda commented:
startbuild

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

number213 commented:
@dsgouda I keep re-running CI job but it keeps failing with Out of Memory exception. Could you please investigate or merge (all of those changes were reviewed when they were merged into network-february-release)?

nschonni commented on specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json:
```diff
@@ -0,0 +1,549 @@
+            "modelAsString": true
+          }
+        },
+        "negationConditon": {
```
Conditon -> Condition

number213 replied:
Server-side typoe, nothing we can do
https://github.com/Azure/azure-rest-api-specs/pull/5341#discussion_r263960243

nschonni replied:
I guess the next API version would fix it then?

number213 replied:
That's a question for @azhao155

dsgouda commented:
@number213 We are working on it.

dsgouda commented:
startbuild

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

dsgouda commented:
@number213 You can ignore the Net Sdk automation error for now, shouldn't block rest of the review.
We will let you know what would be the best course of action for Sdk generation

dsgouda commented:
startbuild

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

dsgouda commented:
@jianghaolu Feel free to merge when ready

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

sergey-shandar committed:
FI from master to network-february-release (#5391)

* Add mixedreality 2019-02-28 API version to Go SDK (#5271)

* Remove ARM gateway Node.js configuration (#5283)

* Updates to Scheduled Query Rule swagger (#5175)

- making aznsaction paramter optional
  - removing hidden-link value from tags
pu

* Fix JavaScript configuration for ARM Private DNS SDK (#5284)

* Fix JavaScript configuration for ARM Private DNS SDK

* Add quotes

* Add private preview API for Azure Image Builder (#5055)

* Add private preview API for Azure Image Builder

* Review comments

 - Added list-by-resource-group and list-by-subscription
 - Added default error response
 - Removed explicit operation url (implied by async contract)
 - Adjusted 'delete' response codes to match expectations
 - Allow patch for tags
 - Added provider operations API

* Review comment: use "nextLink" for paging

* Adding readme files for SDK generation

* PR: rename 'tags' -> 'artifactTags' for distributor

* Add examples, fix minor issues in swagger

* Updating Blueprint spec to include resource group tags (#5263)

* Add Showback model (#5217)

* Add Showback model

* fix ShowbackRule.json example

* remove "," from line 1709

* Add Put Rule

* Fix Scope

* Update json with pull request changes

* change recurring to enum

* chore: Graphservice -> GraphService (#5098)

* Enable SSL on custom domains using users certificate or CDN managed certificate (#5276)

* Translator Text API V3 (#5230)

* Translator Text API V3

* Fix path and override

* Fix a typo and few other improvements in descriptions.

* Remove after_scripts

* Update readme.md

* Update readme.md

* Update MariaDB api version (#5280)

* Adding Swagger and examples for VirtualClusters operation group (#5248)

* Adding Swagger file for VirtualClusters operation group, together with examples.

* Fixing api version, typos and location in examples.

* Adding virtualclusters.json to readme.md

* Security - Updating API versions of alerts and pricing types (#5269)

* Security - Updating api versions of alerts and pricing types

* fix codegen paths for Go SDK

* typo: asscociation -> association (#5252)

* Fix lintdiff issue (#5299)

* fix Go SDK package path for mariadb stable release (#5298)

* changes to improve documentation of endpoint health statuses for routing (#5290)

* add netapp to Go SDK (#5308)

* Update MariaDB default API version to 2018-06-01 (#5302)

* new application insignts api version for unified schema in action (#5292)

* Added Azure App Configuration event data schema. (#5309)

* Added Azure App Configuration event data schema.

* Added a few missing pieces.

* Prefixed event names with our service prefix 'AppConfiguration'.
* Added Microsoft.AppConfiguration to the event grid data-plane readme.

* Updated api-version path to '2018-01-01' from '2018-02-01'.

* Apim Cache resource + DELETE long running + refactor to align with tooling. (#5260)

* api import and service delete long running operation

* Bring your own Cache

align with spec generated from tooling

user and notification align

Refactor products and apis

* align with spec generated from tooling

user and notification align

Refactor products and apis

validating examples

validating examples

move all parameters to apimanagement.json

Fix reference issue with tagcontract and example DeleteCache.

* switch to plural example file names for LIST

* bring back updatehostname and updatecertificate

* Dev adhybridhealthservice microsoft.ad hybrid health service 2014 01 01 binilkk (#5042)

* Fixes for:

Issue #1 : https://github.com/Azure/azure-rest-api-specs/issues/4576#issuecomment-445427819

Typo in the DeltaOperationType Enum.

Issue #2 : https://github.com/Azure/azure-rest-api-specs/issues/4844#event-2014233693

GlobalAdmins list is marked as an object, while it's an array.

* Updated the Specification as example for Risky IP Download report.

* Added the Result details

* Fixed review comments, changed the URI scheme as well as changed get/post methods

* Addressing Review Feedback, changing POST call of blobUri -> generateBlobUri

* Update UserAssignedIdentities spec in Web (#5288)

* Update UserAssignedIdentities spec in Web

Microsoft.Web recently added support for UserAssigned managed service identities (in preview), but the swagger spec is incorrect for the identity payload as it follows the old construct of sending an array of identityIds as opposed to the updated spec which instead sends a dictionary object called userAssignedIdentities. 

Spec + history outlined here:
http://sharepoint/sites/AzureUX/Sparta/_layouts/15/WopiFrame2.aspx?sourcedoc=%7b633632B4-02E4-4DE2-92A9-EDCAF9BE73D4%7d&file=ARM%2BMSI.docx&action=default 

Our swagger contains identityids[] because of previous work to future support UserAssigned identities, but never released the feature in that state, rather we opted to wait for the new spec to be supported and then integrated with that on our end. 

Our MSI implementation reflects Microsoft.Compute's spec for user assigned.

https://github.com/Azure/azure-rest-api-specs/blob/78e1feab142c605839aa3cec23442ca26b8ef04d/specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

* fix linterdiff tool

* Add private dns module name for java codegen (#5312)

* Update Swagger Files For ManagedInstanceAdministrator (#5095)

* Update Swagger Files For ManagedInstanceAdministrator

* Fix for double wording

* Comments resolved

* Bringing consistency between to "ResourceGroup" definition (#5256)

* Bringing consistency between this "ResourceGroup" definition and the one in Common.

* Fixing broken references

* reverted security python readme to pre multi api (#5301)

* [Azure Search] Add Search API to data plane Swagger spec (#5315)

* [Azure Search] Updating some Swagger descriptions to be more doc-friendly

Sometimes we over-describe things, especially "comma-separated lists" in
query string parameters. These are modeled as arrays in Swagger, which are
already understood to be comma-separated lists. They are modeled as
collections in generated code, so they are decidedly not comma-separated
lists in that context. So removing this phrase is a win-win.

* [Azure Search] Adding some missing readOnly attributes

* Adding Search API to the Azure Search data plane spec

* [Azure Search] Use regexes to generate code for server-side paging

This change introduces some very hacky regexes to customize the generated
code for server-side paging of the GET and POST Search APIs. They work by
splitting the generated SearchGet and SearchPost operations in two, so that
the portion that operates on a URL and request body can be reused.

This workaround is necessary because AutoRest lacks flexibility around
paging. The current x-ms-pageable implementation doesn't work for us because
it doesn't model additional top-level response properties or continuation
POST requests (only nextLink/GET).

The only other alternative to this approach besides custom code would have
been to try to parse the nextLink URL query string to reconstitute parameters
to pass to the generated SearchGet and SearchPost operations. This is
unacceptable for Azure Search, since it would exclude custom parameters that
can't be modeled in Swagger, like personalization IDs for custom ranking.
Neither AutoRest nor apparently Open API support modeling arbitrary query
string parameters.

Ideally we'll find a pattern common to other data plane APIs so we don't have
to hack this with regexes for every target language, but it's good enough for
now.

* [Azure Search] Rename some client properties for consistency

* [Azure Search] Make AutocompleteMode non-nullable

* [Hub Generated] Review request for Microsoft.Web to add version 2018-02-01 (#5137)

* Add Hosting Environment network dependencies

* Address PR description feedback

* Add Network Dependencies examples

* Fix casing

* Remove incorrect pagination from examples.

* Add missing MarketplaceOrdering operations to swagger (#5279)

* Add missing MarketplaceOrdering operations to swagger

* Add examples

* Add example files

* Apply suggestions from code review

Clarify operation descriptions

Co-Authored-By: zachraMSFT <zachra@microsoft.com>

* Fix Hosting Environment Network Dependencies pagination (#5320)

* Add regex validation for ManagedClusterAgentPoolProfile.name (#5324)

* Adding instance pools swagger (#5225)

* Initializing instance pools swagger and examples - no usages / managed instances get by instance pool

* updates to use common resource and sku definitions

* Update instance pool swagger with typo fixes

* Updates to instance pool swagger and reverting readme.md to exclude instance pools reference

* Fix whitespace diff

* Making a whitespace change to re-try linter validation

* Updates to add list instances pools by subscription and update license type param to use enum which is limited to LicenseIncluded and BasePrice

* Enhanced A2ARecoveryPointDetails with "disks" (#5307)

* [DiskRP] Adding swagger spec for 09-30-2018 (#5232)

* commit for the older swagger version to make the PR easier to look at

* Adding the DiskRP 09-30-2018 swagger spec. This includes support for hyperVGeneration, encryptionSettingCollections, diskState, Upload Disks, and Write access

* adding the updated readme

* updating the examples to use 09-30-2018

* removing extraneous examples

* adding a description for the EncryptionSettingsElementand fixing linter issues

* [Computer Vision] Add read to CV OCR API. (#5314)

* Add read to OCR API.

* Add description to OCR related JSON properties.

* Mark asyncBatchAnalyze as long running operation.

* Update page property.

* Revert "Mark asyncBatchAnalyze as long running operation."

This reverts commit 51742b3e212c2e900ed5c24f7c351fa7f896d25e.

* added links to limits for application gateway (#5265)

* typo: Winodws -> Windows (#5321)

* MultiApi EventHub (#5331)

* try 2 bootstrapping swagger to .net sdk tools (#5334)

* bootstrapping swagger to .net sdk tools

* deleting nuget.exe binary. Updating configs to locally restore pacakges. Adding the directory that will be used in restoring packages locally to .gitignore

* changing directory

* updating feed location to a readonly directory

* Azure Search: Resource Manager: Support 'None' Identity type (#5319)

* Azure Search: Resource Manager: Support 'None' Identity type

* Azure Search: Resource Manager: Adding serice identity creation\removal examples

* [ADLA] - Catalog - Add stream path to USqlTableFragment (#5329)

* [Hub Generated] Review request for Microsoft.Consumption to add version 2019-01-01 (#5311)

* Add GTM scopes for budgets API

* Removed the extra period at the end of the statement

* fixing stos.proj parameter (#5338)

* change flag name from useUnifiedSchema into useCommonAlertSchema per UX requirement (#5323)

* Enhanced A2A protection input and response details (#5330)

Enhanced A2AEnableProtectionInput & A2AReplicationDetails objects with recoveryAvailabilityZone, recoveryAzureNetworkId, and recoverySubnetName.

* [Hub Generated] Review request for Microsoft.ContainerService to add version 2019-02-01 (#5350)

* Adds base for updating Microsoft.ContainerService from version preview/2018-08-01-preview to version 2019-02-01

* Updates readme

* Updates API version in new specs and examples

* [AKS] add availabilityZones to agent pool profile (#5120)

* including orchestratorVersion and provisioning state for agent pool (#5156)

* including orchestratorVersion and provisioning state for agent pool

* fixing the readme.go.md

* including enablePodSecurityPoliy properties for managedcluster and updating examples

* fixing the example for enablePodSecurityPolicy

* including apiServerAuthorizedIPRanges (#5286)

* including apiServerAuthorizedIPRanges

* support azure network policy

* mark the new features as PREVIEW in description  (#5310)

* mark the new features as PREVIEW

* mark vmss as required for az

* Azure Event Grid: Add 2019-02-01-preview tags for Ruby and Go readme files (#5349)

* Add 2019-02-01-preview tags for Ruby and Go readme files

* fix wrong preview subdirectory

* Update some descriptions. (#5305)

* update description. (#5335)

* Added VNet API (#5351)

* EventHub : corrected typo  (#5360)

* corrected typo in uri

* removed unused parameters

* Add annotations property to trigger (#5332)

* EventHub: Revert the NetworkRuleSet  (#5363)

* Revert "EventHub : corrected typo  (#5360)"

This reverts commit 1495122039e95d8795e3769d06c60e290cb53d2b.

* Revert "Added VNet API (#5351)"

This reverts commit f2ab4a8208bd3de1813b26b078429357dbd05506.

* Remove the AzureDataLakeStoreCosmosStructuredStreamDataset since it's internal only (#5277)

* Removing SDK generation from Travis-CI (#5367)

* Microsoft.DataBoxEdge RP: Corrected comments of trigger and extendedinfo APIs, corrected examples (#5322)

* Corrected comments of trigger and extendedinfo APIs, corrected examples

* Microsoft.DataBoxEdge RP: Have better examples

* Corrected parameter description in order API

* oad => @azure/oad (#5362)

* oad => @azure/oad

* oad => @azure/oad

* SRP Swagger changes for 2018-11-01 (#5300)

* SRP Swagger changes for 2018-11-01

* Address CR comments

* Restrict ManagementPolicyRule -> type property to have only values from defined enum

* Make ManagementPolicySchema/version an enum string

* Remove 'version' property from ManagementPolicy schema

* Added List Policies API to SecurityAlerts (#5070)

* Added SecurityAlert List API

Added List api to:
Manged Server + Managed Database
Logical Server + Logical Database

* FIxed a description

this property is for databases+servers

* Added missing Database exmaples

Added missing examples for database security alert

* Added missing description change

For Server Security Alerts

* Revert "FIxed a description"

This reverts commit e01a332d6e97ebedf04677ea5eb356ae4570d97b.

* Revert "Added missing description change"

This reverts commit 17aeab7011d6085d08e562248eb89c2b2fe40277.

* Update DatabaseSecurityAlertListByDatabase.json

* fixed examples

* Update DatabaseSecurityAlertCreateMin.json

Deleted wild comma

* Update readme.md

* Minor fixes

Changed tabs to spaces.
Added location for servers exmaples.

* Removed location

removed the location property from examples

* graph: improve API coverage on application & service principal  (#5181)

* [graphrbac]: fix bugs in permission grant apis

* fix paging

* add missing property

* remove pageable

* author pagable in the right way

* breaking changes over a wrong spec

* remove additionalProperties which causes wrong code-gen

* add suppressions

* use code to surpress

* address review feedabck

* moev ahead with the established pattern anyway

* address review feedbacks

* clean up all

* update operation id

* add one more property per CLI user's ask

* Add insights 2019-03-01 package to Go SDK (#5373)

* Add insights 2019-03-01 package to Go SDK

* move global settings before packages

* fix package tag for new API version (#5378)

* Remove GZRS and RAGZRS sku from SRP 2018-11-01 because feature slipped. (#5377)

* typo: upto -> up to (#5376)

* typo: graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6 (#5375)

- requied -> required
- authorizaed -> authorized
- Organizatio -> Organization
- Allorganizations -> AllOrganizations
- Microsof -> Microsoft
- perperties -> prorperties
- servie -> service
- altenative -> alternative
- thubmbprint -> thumbprint

* Azure DevOps Pipeline workaround for private repository (#5374)

* Fix PolicyInsights code generation issue (#5389)

* Add replica creation parameters to Postgres ResourceProvider for SDK (#5379)

*  Add replica creation parameters to Postgres ResourceProvider for SDK

* Add replica creation parameters to Postgres ResourceProvider for SDK

* Add replica creation parameters to Postgres ResourceProvider for SDK

* Add replica creation parameters to Postgres ResourceProvider for SDK

* Add replica creation parameters to Postgres ResourceProvider for SDK

sergey-shandar committed:
Revert "FI from master to network-february-release (#5391)" (#5393)

This reverts commit 8f28f7f041ff15ad42c21f1a342cdee8f35ecb4c.

sergey-shandar committed:
Merge pull request #5394 from Azure/master

FI from master to network-february-release

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

dsgouda commented:
@jianghaolu @number213 Please ignore the Net Sdk generation failure and merge this PR if ready

dsgouda reviewed (approved):
LGTM

adxsdknet commented:
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/-1 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/-1/commits/

jhendrixMSFT commented:
@AutorestCI regenerate azure-sdk-for-go
